catkin_ws/src/state_estimation/launch/state_estimation.launch

```diff
@@ -26,7 +26,7 @@
     </group>
     <group if="$(arg sim)">
 		<param name="update_state_on_clock" value="true" />
-		<node name="state_aggregator" pkg="state_estimation" type="state_aggregator" respawn="true" output="screen"/>
+		<node name="state_aggregator" pkg="state_estimation" type="state_estimation" respawn="true" output="screen"/>
```

Comment:
how does this work? (genuine question) I thought that .cpp files in state estimation would be compiled as executables and then we run those, so how is state_estimation an executable which is built?

Reply:
in the cmakelists add_executable function i just set it to the project name variable instead of naming it something special

Reply:
think of how you compile with gcc. 
```bash
gcc a.cpp -o b
```
the resulting executable can be anything. its the same in cmake. instead of requesting it be called state_aggregator i just passed it the value of the project variable which is fairly standard i believe

Reply:
ah ok i didnt realize u had set that in cmake gotcha thanks